app/assets/stylesheets/govuk_publishing_components/components/_contents-list.scss

```diff
@@ -49,17 +49,17 @@
 }
 
 .gem-c-contents-list__list-item {
-  padding-top: $gutter-one-third;
+  padding-top: govuk-spacing(2);
   line-height: 1.3;
   list-style-type: none;
 
-  @include media(tablet) {
-    padding-top: $gutter-one-quarter;
+  @include govuk-media-query($from: tablet) {
+    padding-top: govuk-spacing(6) / 4;
   }
 }
 
 .gem-c-contents-list__list-item--dashed {
-  $contents-spacing: $gutter-half + 10;
+  $contents-spacing: govuk-spacing(3) + govuk-spacing(2);
```

Comment:
Not sure if possible, but perhaps these can be squished together now?

Reply:
Yep, they can..  was looking to see if there's any reason of them being split (such as making it more obvious where the value comes from, but it doesn't seem to be the case.